graph/docu/docu_04graph_02plotutils.cc

```diff
@@ -67,7 +67,9 @@ for (size_t e = 0; e < 2; ++e) {
     .OutputFormat("svg");
 #endif
 if (FLAGS_regenerate_golden_graphs) bricks::FileSystem::WriteStringToFile(result, (std::string("golden/love.") + extensions[e]).c_str());
+#ifndef BRICKS_APPLE // TODO(dkorolev): Figure out how to run this test on Apple.
 if (!e) ASSERT_EQ(result, bricks::FileSystem::ReadFileAsString(std::string("golden/love.") + extensions[e]));
```

Comment:
It's not about installing the tool or its version, it's about the inconsistency of its output binary-wise.

If only we had a simple way to do a visual diff, it would be resolved. But I am not aware of such a way for Linux+Mac+Windows.

We'll likely go with an SVG with non-image parts `grep`-ped out. Stay tuned.

And I'd say the code is fine for now. The images library is not critical for Bricks.